deploy/openshift/deploy_che.sh

```diff
@@ -65,7 +65,7 @@ wait_until_che_is_available() {
     DEPLOYMENT_TIMEOUT_SEC=300
     POLLING_INTERVAL_SEC=5
     end=$((SECONDS+DEPLOYMENT_TIMEOUT_SEC))
-    while [ "${available}" != "\"True\"" ] && [ ${SECONDS} -lt ${end} ]; do
+    while [[ "${available}" != "\"True\"" || "${progressing}" != "\"True\"" ]] && [ ${SECONDS} -lt ${end} ]; do
```

Comment:
It would look simpler if we replace 
`[[ "${available}" != "\"True\"" || "${progressing}" != "\"True\"" ]]`
on
`! [[ "${available}" == "\"True\"" && "${progressing}" == "\"True\"" ]]`

Reply:
Or even simpler 
`while ! [ "${available}" == "\"True\"" ] && ! [ "${progressing}" == "\"True\"" ] && [ ${SECONDS} -lt ${end} ]; do`

Reply:
Personally, I think it wouldn't. For me easier to understand local negation but not complex one.

Reply:
@dmytro-ndp you can propose another PR with improvement but please consider all places with such codestyle.

Reply:
From my experience, a negation of a set of expressions is harder for understanding and fast reading of the code than several negations of simple and small expressions. So I opt to leave it as is

Reply:
> Or even simpler
> ! [ "${available}" == "\"True\"" ] && ! [ "${progressing}" == "\"True\"" ] && [ ${SECONDS} -lt ${end} ]

I agree that it is simpler. But I'm afraid that it works differently from original if statement
I mean that you can't transform `(!A || !B) & C` to `!A & !B & C`

Reply:
@garagatyi: 
> several negations of simple and small expressions
that is what I has proposed as second variant: 
`! [ "${available}" == "\"True\"" ] && ! [ "${progressing}" == "\"True\"" ] && [ ${SECONDS} -lt ${end} ]`

Reply:
@sleshchenko: my fault. You are right.